solr/solr-ref-guide/modules/indexing-guide/examples/IndexingNestedDocuments.java

```diff
@@ -66,89 +69,94 @@ public static SolrClient getSolrClient() {
    */
   public void testIndexingAnonKids() throws Exception {
     final String collection = "test_anon";
+
     CollectionAdminRequest.createCollection(collection, ANON_KIDS_CONFIG, 1, 1)
         .setPerReplicaState(SolrCloudTestCase.USE_PER_REPLICA_STATE)
         .process(cluster.getSolrClient());
-    cluster.getSolrClient().setDefaultCollection(collection);
+
+    // configure the client with the default collection name, to simplify our example below.
+    IndexingNestedDocuments.clientUsedInSolrJExample =
+        cluster.basicSolrClientBuilder().withDefaultCollection(collection).build();
 
     //
-    // DO NOT MODIFY THESE EXAMPLE DOCS WITH OUT MAKING THE SAME CHANGES TO THE JSON AND XML
-    // EQUIVILENT EXAMPLES IN 'indexing-nested-documents.adoc'
+    // DO NOT MODIFY THESE EXAMPLE DOCS WITHOUT MAKING THE SAME CHANGES TO THE JSON AND XML
+    // EQUIVALENT EXAMPLES IN 'indexing-nested-documents.adoc'
```

Comment:
Did that happen?

Reply:
Yeah, as in, it didn't need to be done!  Thanks for the reminder!   The code change was actually just typo fixes.